app/views/hyrax/dashboard/collections/_form_relationships.html.erb

```diff
@@ -43,46 +34,27 @@
   <section>
     <h3><%= t('.headlines.other_collections_in_this_collection') %></h3>
     <p><%= t('.add_other_collections_as_sub_collections_description') %></p>
-
-    <article class="form-inline select-wrapper">
-      <div class="form-group">
-        <label for="exampleInputName2"><strong><%= t('.add_sub_collection') %>:</strong></label>
```

Comment:
Remove from i18n?

Reply:
Good catch. I guess I didn't verify all of the related deletions triggered by all of my deleted code. I'll dig into that along with verifying the partials to remove from the TODO I had noted.